### ISSUE TITLE
Fails to operate on a parameter that is ','

### ISSUE THREAD

**Issue by h3xx** [sgur/vim-textobj-parameter#3]
When a parameter is `','` the plugin fails to operate on it when the cursor is in a certain position in the textobject.

Test code:

```
<?php
implode(',', $foo);
#        ^ put cursor block ON the comma
```

Here's the tests I ran when typing `da,` where the cursor is shown.
- Green is expected behavior.
- Red is broken behavior.
- Blue means it fails to operate on the parameter, but that's to be expected.

![screenshot-2016-0308-1112---](https://cloud.githubusercontent.com/assets/615684/13609692/b68d8496-e51f-11e5-9ce3-61a8ee49661c.png)

**Comment by sgur** [comment]
Thank you for reporting.
I was confirmed the problem. I'll fix it.

**Comment by sgur** [comment]
This issue has been fixed at 184dd1b4 .

**Comment by h3xx** [comment]
Confirmed fixed. Thanks! :-)
